practice-typescript/src/scripts/constants/emptyTable.ts

```diff
@@ -0,0 +1,7 @@
+enum EMPTY_MESSAGE {
```

Comment:
It's not good @thanhtrantruong47. USER_NOT_FOUND doesn't mean the user list is empty. So why you put it in the EMPTY_MESSAGE

Reply:
fixed